addons/website_sale/static/src/xml/website_sale_utils.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+<!-- Products Search Bar autocomplete item -->
+<div t-name="website_sale.productsSearchBar.autocompleteItem">
+    <a t-att-href="item.url">
+        <div class="media o_search_product_item">
+            <t t-if="products_options.display_image and item.image">
+                <img class="rounded mr-3" t-att-src="'data:image/png;base64,' + item.image"/>
+            </t>
+            <div class="media-body">
+                <h6 t-att-class="item.description ? 'mt-2' : 'mt-3'">
+                    <t class="o_product_info" t-esc="item.name"/>
+                    <t t-if="products_options.display_price">
+                        <b><span class="float-right" t-esc="price_with_currency"/></b>
+                    </t>
+                </h6>
+                <t t-if="products_options.display_description and item.description">
+                    <t class="o_product_info" t-esc="item.description"/>
```

Comment:
t class does nothing

Reply:
What do you mean ? :smile:

Reply:
well... that you put a class on a t element... which is not a DOM element... so this class will not appear.